src/main/java/io/phasetwo/service/resource/OrganizationsResource.java

```diff
@@ -68,13 +70,31 @@ public Response me() {
   @Path("")
   @Produces(MediaType.APPLICATION_JSON)
   public Stream<Organization> listOrgs(
-      @QueryParam("search") String searchQuery,
+      @QueryParam("search") String search,
       @QueryParam("first") Integer firstResult,
-      @QueryParam("max") Integer maxResults) {
-    log.debugf("listOrgs %s %s %d %d", realm.getName(), searchQuery, firstResult, maxResults);
+      @QueryParam("max") Integer maxResults,
+      @QueryParam("q") String searchQuery) {
     firstResult = firstResult != null ? firstResult : 0;
     maxResults = maxResults != null ? maxResults : Constants.DEFAULT_MAX_RESULTS;
-    return orgs.searchForOrganizationByNameStream(realm, searchQuery, firstResult, maxResults)
+
+    if (search != null && searchQuery != null) {
```

Comment:
@phamann Thinking more about this as I update the java client. Would there be a use case where the caller would expect if both were present that it should search on both an attribute **and** a name?

Reply:
Thanks for doing that @phamann . We are aware of some customer use cases where they would probably use both.